src/ci/docker/arm-android/Dockerfile

```diff
@@ -53,4 +53,8 @@ ENV RUST_CONFIGURE_ARGS \
 # to all the targets above eventually.
 ENV SCRIPT \
   python2.7 ../x.py test --target arm-linux-androideabi && \
-  python2.7 ../x.py dist --target arm-linux-androideabi
+  python2.7 ../x.py dist \
+      --target arm-linux-androideabi \
+      --target armv7-linux-androideabi \
+      --target i686-linux-android \
+      --target aarch64-linux-android
```

Comment:
Is it reasonably easy to rename this directory to just 'android' since it's not arm-only now?